sdk/storage/perf-test/src/main/java/com/azure/storage/blob/perf/UploadBlockBlobTest.java

```diff
@@ -3,24 +3,33 @@
 
 package com.azure.storage.blob.perf;
 
-import com.azure.perf.test.core.RandomFlux;
-import com.azure.perf.test.core.RandomStream;
-import com.azure.perf.test.core.SizeOptions;
+import static com.azure.perf.test.core.TestDataCreationHelper.createRandomByteBufferFlux;
+import static com.azure.perf.test.core.TestDataCreationHelper.createRandomInputStream;
+
+import com.azure.perf.test.core.PerfStressOptions;
 import com.azure.storage.blob.perf.core.BlobTestBase;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
-public class UploadBlockBlobTest extends BlobTestBase<SizeOptions> {
-    public UploadBlockBlobTest(SizeOptions options) {
+public class UploadBlockBlobTest extends BlobTestBase<PerfStressOptions> {
+    private final InputStream randomInputStream;
```

Comment:
Could a single `InputStream` or `Flux<ByteBuffer>` be used for all test instances?  If so these should be static.

Reply:
Agree. Will make this static,

Reply:
Just realized that to create this input stream and flux, we need `PerfStressOptions` which is per-instance. So, can't make this static.